spec/requests/outgoings_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+RSpec.describe OutgoingsController, type: :request do
+  describe 'POST /assessments/:assessment_id/outgoings' do
+    let(:assessment) { create :assessment }
+    let(:payment_date) { 3.weeks.ago.strftime('%Y-%m-%d') }
+    let(:outgoings) { attributes_for_list(:outgoing, 2, payment_date: payment_date) }
+    let(:headers) { { 'CONTENT_TYPE' => 'application/json' } }
+
+    let(:params) do
+      {
+        outgoings: outgoings
+      }
+    end
+
+    subject { post assessment_outgoings_path(assessment), params: params.to_json, headers: headers }
+
+    it 'returns http success', :show_in_doc do
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'creates outgoings' do
+      expect { subject }.to change { Outgoing.count }.by(2)
```

Comment:
`expect { subject }.to change { assessment.outgoings.count }.by(2)` would be better

Reply:
Yup - did that in the service spec. Missed it here. Updated.